src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  --color-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo__link {
+  cursor: pointer;
+  display: flex;
```

Comment:
why do we need `display: flex` here? just add height style

Reply:
done

Reply:
done